### PR TITLE
fix-181: changed the welcome text on the install page

### PR DESCRIPTION
**Pull request by yatsenko-julia**
## Description

I've changed the welcome text on the install page. 

## Motivation and context

Here is [issue-181](https://github.com/ClassicPress/ClassicPress/issues/181).

## Types of changes

- [x] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: src/wp-admin/install.php [hunk 1]
```diff
@@ -315,7 +315,7 @@ function display_setup_form( $error = null ) {
 		display_header();
 ?>
 <h1><?php _ex( 'Welcome', 'Hello' ); ?></h1>
-<p><?php _e( 'Welcome to the famous five-minute ClassicPress installation process! Just fill in the information below and you&#8217;ll be on your way to using the most extendable and powerful personal publishing platform in the world.' ); ?></p>
+<p><?php _e( 'Welcome to the ClassicPress installation process! ClassicPress brings all of the latest features the best modern CMS while keeping the old and proven way to compose and manage your content. Just fill in the information below and you&#8217;ll be on your way to using the most extendable and powerful personal publishing platform in the world.' ); ?></p>
```
Comment (senlin) [inline src/wp-admin/install.php]: It seems there is a word missing in:

"ClassicPress brings all of the latest features the best modern CMS while keeping..."

**Comment by senlin** [comment]
After approving your change I noticed that at the end of the sentence, we most likely also need another change:

"... you&#8217;ll be on your way to using the most extendable and powerful personal publishing platform in the world." As ClassicPress targets businesses, we might need to change that into something more business-like?

**Comment by yatsenko-julia** [comment]
@nylen 
I thought that ClassicPress was essentially a WordPress without their new Gutenberg page composer, so everything that WordPress has and will have, the ClassicPress will have also? However, I have changed the working so it does not stress the “all of the latest features”. Let me know if it sounds better to you.

**Comment by yatsenko-julia** [comment]
@senlin 
Right, I’ve missed the “personal publishing platform” bit. I’ve taken a stab at rehearsing it, could you please let me know if it feels better.

**Pull request merged**